packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -342,7 +342,7 @@ $bpk-spacing-v2: true;
     bpk-spacing-md() center;
   background-size: $bpk-input-height / 2;
   color: $bpk-select-color;
-  line-height: $bpk-line-height-base - (2 * $bpk-select-border-width);
+  line-height: ($bpk-line-height-base - (2 * $bpk-select-border-width));
```

Comment:
The linter through an error when I made the further down change with the following so to fix it surrounded it with `()`

<img width="1191" alt="Screenshot 2021-12-13 at 13 17 08" src="https://user-images.githubusercontent.com/8831547/145819427-d41f8489-8b50-4283-89b3-3b8596c44375.png">